api/src/main/java/uk/gov/dhsc/htbhf/claimant/reporting/payload/ReportPropertiesFactory.java

```diff
@@ -95,17 +100,29 @@ protected Long calculateQueueTime(LocalDateTime timestamp) {
         customDimensions.put(WESTMINSTER_PARLIAMENTARY_CONSTITUENCY.getFieldName(), postcodeData.getParliamentaryConstituency());
         customDimensions.put(CLINICAL_COMMISSIONING_GROUP.getFieldName(), postcodeData.getCcg());
         customDimensions.put(CLINICAL_COMMISSIONING_GROUP_CODE.getFieldName(), postcodeData.getCodes().getCcg());
-        customDimensions.put(QUALIFYING_BENEFIT.getFieldName(), context.getIdentityAndEligibilityResponse().getQualifyingBenefits());
+        customDimensions.put(QUALIFYING_BENEFIT.getFieldName(), getQualifyingBenefits(identityAndEligibilityResponse));
 
         return customDimensions;
     }
 
+    private List<LocalDate> getDobOfChildrenUnder4(Claim claim, CombinedIdentityAndEligibilityResponse identityAndEligibilityResponse) {
+        return identityAndEligibilityResponse == null
+                ? claim.getClaimant().getInitiallyDeclaredChildrenDob()
+                : identityAndEligibilityResponse.getDobOfChildrenUnder4();
+    }
+
+    private QualifyingBenefits getQualifyingBenefits(CombinedIdentityAndEligibilityResponse identityAndEligibilityResponse) {
+        return identityAndEligibilityResponse == null
+                ? QualifyingBenefits.NOT_SET
+                : identityAndEligibilityResponse.getQualifyingBenefits();
+    }
+
     protected Map<String, Object> createCommonCustomMetrics(ReportEventMessageContext context) {
         Map<String, Object> customMetrics = new TreeMap<>();
         Claimant claimant = context.getClaim().getClaimant();
         LocalDate atDate = context.getTimestamp().toLocalDate();
         customMetrics.put(CLAIMANT_AGE.getFieldName(), Period.between(claimant.getDateOfBirth(), atDate).getYears());
-        List<LocalDate> dobOfChildrenUnder4 = context.getIdentityAndEligibilityResponse().getDobOfChildrenUnder4();
+        List<LocalDate> dobOfChildrenUnder4 = getDobOfChildrenUnder4(context.getClaim(), context.getIdentityAndEligibilityResponse());
```

Comment:
Can context.getClaim be declared as a variable instead as it's now used more than once